### PR TITLE
chore: Add title back into Readme

### PR DESCRIPTION
**Pull request by fhinkel**
The Readme should have a title, especially obvious if it's missing for Cloud Rad.

### CODE REVIEW

**Review by JustinBeckwith — COMMENTED** [review]
There's a typo in introductin

File: .readme-partials.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-title: |-
```
Comment (dandhlee) [inline .readme-partials.yml:1]: Just slightly confused, does removing the title here add it somewhere else?

Comment (fhinkel) [reply .readme-partials.yml:1]: I believe the previous version overwrites title with an empty title. Removing `title` actually adds the title in. But really I'm just following the other repos' partial files where the title displays correctly.

Comment (dandhlee) [reply .readme-partials.yml:1]: Sounds good! Thank you :)

**Review by dandhlee — APPROVED** [review]
LGTM.

**Comment by fhinkel** [comment]
Fixed the typo.

**Review by JustinBeckwith — APPROVED** [review]

**Review by bcoe — APPROVED** [review]

**Pull request merged**